ios/MattermostShare/ChannelsViewController.swift

```diff
@@ -0,0 +1,134 @@
+import UIKit
+
+class ChannelsViewController: UIViewController {
+
+  let searchController = UISearchController(searchResultsController: nil)
+  
+  lazy var tableView: UITableView = {
```

Comment:
You never mentioned to us that Swift had lazy evaluation! That's pretty nice to have that provided automatically.

Reply:
Yes swift seems to be very powerful, I'm sure there are a lot of other stuff that we are not using that could be incredible.